### DISCUSSION TITLE
Total value for a specific parameter

### DISCUSSION THREAD

**Discussion by albertan352652** [DaxStudio/DaxStudio#636] [Q&A] [answered]
Hello, I appreciate if I get some help.

I connected power pivot model to Dax Studio. I just want to see the total value (sum of the amount as one number) for a specific project number. How can I do that?
When I try to use Query Builder it gives me several lines but I want only one value. 
is it ok to use CALCULATETABLE(
{[Amount]},
Project = "ABC"
At times I'm getting an error message says that I need to use SUM or other formula if I want to get one amount

Or is there a better way to do it perhaps through Query Builder?

Thanks

**Comment by dgosbell** [comment, marked as answer]
If you want to aggregate a column you would typically use a measure. But this is an interesting question, because one feature we don't have in the query builder is the option to generate implied measures on the fly from numeric columns like Power Pivot and Power BI can. I might see if there is a way I can add this in a future release, but for the time being there are a couple of possible solutions

1. In the Query Builder you could start by clicking the "New" button in the top right. This will open a screen where you can create a new (temporary) measure that only exists within the current query and if you just enter something like `SUM( '<table_name>'[<column_name>] )` then add the filter you want to the filter section this should give you a single figure as an output

2. If you want to write the query by hand something like the following would also work

```
EVALUATE
{
  CALCULATE (  
      SUM( '<table_name>'[<column_name>] ),
      '<table_name>'[Project] = "ABC"
  )
}
```

_Note: you would need to replace the <table_name> and <column_name> references with the actual names from your model_
